### ISSUE TITLE
Focus hardly noticeable on the Edit Profile page

### ISSUE THREAD

**Issue by rpkoller** [AzuraCast/AzuraCast#6681]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

#d64b1a4 (2023-10-20 0:01)

### What happened?

The color contrast for the focus outline for the different interface components isn't satisfy wcag 2.2 SC 2.4.11 (https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum)

![Screenshot 2023-10-20 at 03 55 46](https://github.com/AzuraCast/AzuraCast/assets/1665422/ec3b6db1-12a1-4cb2-be37-8093e7d224f0)

close button: #CCE2F9 1,3:1
For the field and radio button: #8FC4F2. 1.9:1

=> all three fail the minimum color contrast. in addition field and radio button seem one pixel thin only. recommendation is at least 2 pixel 

close button
#C0C4C7 1.8:1 against white background 
#C0C4C7 3.7:1 against #575E64  button background

=> even though one half has enough color contrast and the focus width is also broad enough personally i have a hard time to notice that there is a focus outline at all. for ee the dark close button background color is way too dominant so the lightish grey drown. 

save changes button:  
#A4CCF5. 1.7:1 against white background 
#1A74C3. 2.9:1 against #A4CCF5 button background
=> similar to the close button one half close to a good enough color contrast but visually really hard to recognize that there is a focus outline at all. and even for people with not visual issue at all try to run azurecast from your tablet on the veranda or the porch in the bright sun you will have a hard time recognizing the element in focus that way.

**Comment by rpkoller** [comment]
and one addition after finishing the write up of the three issue i remembered that there is also the windows high contrast mode. i've quickly tested in edge. the interface without interaction looks fab. everything visible. but in the context to focus visibility there is an issue therefore i add it as a comment in here. if you tab through the interface in edge the focus is not visible when the one tab group is in focus. regular tab time display shift tab language.

**Comment by rpkoller** [comment]
high contrast mode in edge is available on the rendering tab in the bottom of devtools. there you check enable automatic dark mode and set emulate css media feature forced color to active and emulate css media feature prefers contrast to more .

**Comment by BusterNeece** [comment]
I updated some of the variables used by Bootstrap to generate the button and component focus states and significantly improved their visibility. I'm not sure if it's WCAG approved or anything, but if you look at it versus the previous iteration, it's significantly easier to tell what's focused.

**Comment by rpkoller** [comment]
i've checked the latest changes. the focus for the buttons got more apparent. at least i am able to spot a change but that isnt due to the focus outline but more due to the change of the background color of the button. the outline of the fields remains more or less the same visually for me and the selected radio buttons are also not such an apparent change. but as mentioned in my other comment due to my visual conditions i am a good human litmus test for that kind of color contrast issues. and again like for the color contrast issue there is also the dark mode i had forgot. 

but on the other hand it doesnt make sense to fix every a11y styling related issue in the azuracast theme which is based on bootstrap. better if it would be fixed upstream otherwise it becomes a sisyphus task :/ skimmed through the bootstrap repo. there is for example https://github.com/twbs/bootstrap/issues/29422 patrick lauke chimed in there and left a few comments in regards of focus outlines in particular. so something seems in the works for a while. but getting that color contrast stuff for text, none text and focus well aligned that it is accessible and also looks good plus have that for a light AND a dark mode is a challenging task :/ 

and i took a closer look in the current implementation in azuracast there is a outline:0 and a box shadow. that leads to that thin 1 px outline and a lighter three pixel shadow
 
![Screenshot 2023-10-21 at 13 41 57](https://github.com/AzuraCast/AzuraCast/assets/1665422/7c530425-5af8-4782-9187-39aa18991c4d)

if you comment out the outline and the boxshadow in the devtools the color contrast is still too low but due to the solid and thick outline it is at least more clearly visible. but definitely a none trivial problem :/
